src/components/general/SideSheet/Standard/index.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { FC, useState, useCallback, useRef, useEffect, ReactNode } from 'react';
+import classNames from 'classnames';
+import { useComponentDisplayClassNames } from '@equinor/fusion';
+import { useEventListener } from '@equinor/fusion-components';
+import styles from './styles.less';
+import CollapseExpandButton from './CollapseExpandButton';
+import { OverlayPortal, Scrim } from '@equinor/fusion-components';
+
+export type StandardSideSheetProps = {
+    id: string;
+    title?: string;
+    isOpen: boolean;
+    onOpenChange: (isOpen: boolean) => void;
+    screenPlacement?: 'right' | 'left';
+    children: ReactNode;
+};
+
+const SideSheet: React.FC<StandardSideSheetProps> = ({
+    id,
+    title,
+    isOpen,
+    onOpenChange,
+    children,
+    screenPlacement = 'right',
+}) => {
+    const toggleOpen = useCallback(() => {
+        onOpenChange(!isOpen);
+    }, [isOpen]);
+
+    const containerClassNames = classNames(
+        styles.container,
+        useComponentDisplayClassNames(styles),
+        {
+            [styles.isCollapsed]: !isOpen,
+            [styles.screenPlacementLeft]: screenPlacement === 'left',
+        }
+    );
+
+    return (
+        <div className={containerClassNames} key={id}>
+            <div className={styles.header}>
+                <div className={styles.collapseButtonContainer}>
+                    <CollapseExpandButton
+                        isOpen={isOpen}
+                        onClick={toggleOpen}
+                        screenPlacement={screenPlacement}
+                    />
+                </div>
+                {isOpen && title && <div>{title}</div>}
```

Comment:
Maybe the title should have som styling?

Reply:
Styled title